packages/portal/src/pages/account/api-keys.vue

```diff
@@ -69,9 +78,6 @@
                       </template>
                     </template>
                   </b-table>
-                  <p v-else>
-                    {{ $t('apiKeys.noKeys') }}
```

Comment:
If no longer needed, remove the key from the i18n file.

Reply:
That's unused and removed.